networkx/classes/function.py

```diff
@@ -787,6 +787,10 @@ def set_edge_attributes(G, values, name=None):
         >>> G[1][2]["attr2"]
         3
 
+    The attributes of one Graph can be used to set those of another.
+        >>> H = nx.path_graph(3)
+        >>> nx.set_edge_attributes(H, G.edges)
+
```

Comment:
```suggestion
    The attributes of one Graph can be used to set those of another::
    
        >>> H = nx.path_graph(3)
        >>> nx.set_edge_attributes(H, G.edges)

```

Just for consistency with the other examples.